engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/BpmnEventTypeTest.java

```diff
@@ -322,6 +322,88 @@ void executeInstance() {
                   .withCorrelationKey(CORRELATION_KEY)
                   .publish();
             }
+          },
+          new BpmnEventTypeScenario(
+              "Escalation Start Event", BpmnElementType.START_EVENT, BpmnEventType.ESCALATION) {
+            @Override
+            BpmnModelInstance modelInstance() {
+              return Bpmn.createExecutableProcess(processId())
+                  .eventSubProcess(
+                      "subprocess",
+                      sp -> sp.startEvent(elementId()).escalation(escalationCode()).endEvent())
+                  .startEvent()
+                  .intermediateThrowEvent("throw", i -> i.escalation(escalationCode()))
+                  .endEvent()
+                  .done();
+            }
+
+            @Override
+            void executeInstance() {
+              super.executeInstance(Collections.emptyMap());
+            }
+          },
+          new BpmnEventTypeScenario(
+              "Escalation Throw Event",
+              BpmnElementType.INTERMEDIATE_THROW_EVENT,
+              BpmnEventType.ESCALATION) {
+            @Override
+            BpmnModelInstance modelInstance() {
+              return Bpmn.createExecutableProcess(processId())
+                  .eventSubProcess(
+                      "subprocess",
+                      sp -> sp.startEvent("catch", s -> s.escalation(escalationCode()).endEvent()))
+                  .startEvent()
+                  .intermediateThrowEvent(elementId(), e -> e.escalation(escalationCode()))
+                  .endEvent()
+                  .done();
+            }
+
+            @Override
+            void executeInstance() {
+              super.executeInstance(Collections.emptyMap());
+            }
+          },
+          new BpmnEventTypeScenario(
+              "Escalation Catch Event", BpmnElementType.BOUNDARY_EVENT, BpmnEventType.ESCALATION) {
```

Comment:
🔧 We should name this scenario "Escalation Boundary Event". I noticed we also have the wrong name for the error event. Would you mind renaming that one as well?